packages/logger/__tests__/unit/consolaStructuredLogger.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { createConsolaStructuredLogger } from "../../src/ConsolaStructuredLogger";
+import type { StructuredLog } from "../../src/interfaces/StructuredLog";
+import type { LogData, Metadata } from "../../src/interfaces/LoggerInterface";
+
+process.env.NODE_ENV = "development";
+
+describe("createConsolaStructuredLogger", () => {
+  const mockStructuredLog: StructuredLog = {
+    createLog: jest.fn(),
+  };
+
+  const logData: LogData = "Test log message";
+  const metadata: Metadata = { userId: "12345" };
+
+  it("should create a logger with the correct set of methods", () => {
```

Comment:
nit pick: I usually find `should` in the description of tests redundant. `It creates`, It logs` would be enough

Reply:
ok